resources/styles/global/forms.less

```diff
@@ -1,3 +1,5 @@
+@form-error-color: #E35C3B;
```

Comment:
Should this be defined in `variables.less` or already in `bootstrap-material-design.less`?